pootle/apps/pootle_app/management/commands/__init__.py

```diff
@@ -76,21 +69,6 @@ def do_translation_project(self, tp, pootle_path, **options):
                 logging.exception(u"Failed to run %s over %s's files",
                                   self.name, tp)
                 return
-        elif hasattr(self, "handle_store"):
```

Comment:
I don't think removing this whole branch is what you want. You'll only want to remove the `if pootle_path:` branch below; there are important commands implementing `handle_store`.